config/routes.rb

```diff
@@ -1,11 +1,9 @@
 Rails.application.routes.draw do
+  resources :recipes, only: [:index, :show]
+
   get 'ingredients/index'
 
   root 'ingredients#index'
 
-  get 'recipes/show'
```

Comment:
I read it twice, but still I dont know what should I put in 'application.html.erb' layout in navbar to redirect page to the view of one specific recipe, which is "/views/recipes/show". I tried to use sth like: <%= link_to "Random!", recipes_path(recipe)%>, but it doesnt work. The site works when I add the line removed by Sandra, though :(